webknossos/tests/dataset/test_dataset.py

```diff
@@ -867,6 +867,7 @@ def test_chunking_wk(data_format: DataFormat, output_path: Path) -> None:
     ds_path = prepare_dataset_path(data_format, output_path)
     ds = Dataset(ds_path, voxel_size=(2, 2, 1))
     chunk_shape, chunks_per_shard = default_chunk_config(data_format, 8)
+    shard_shape = chunk_shape * chunks_per_shard
```

Comment:
I think the naming of `shard_shape` and `chunk_shape` is confusing. Why is the `chunk_shape` for initializing the mag different from the `chunk_shape` for iterating over the mag?

Reply:
Yes, that is a bit unfortunate. shard and chunk are the terms of the Zarr storage, but chunk is also used for the iterator functions. Do you have an idea how to resolve that?